cmd/compose/run.go

```diff
@@ -133,6 +134,15 @@ func runCommand(p *ProjectOptions, streams api.Streams, backend api.Service) *co
 				}
 				opts.entrypointCmd = command
 			}
+			if cmd.Flags().Changed("tty") {
+				if cmd.Flags().Changed("no-TTY") {
+					if opts.tty != !opts.noTty {
```

Comment:
This breaks my brain 😂 (booleans are such a pain)

Should we make it "irregardless of what is passed" and always make it conflict?

```go
if cmd.Flags().Changed("no-TTY") && cmd.Flags().Changed("tty") {
    return fmt.Errorf("--tty and --no-TTY can't be used together")
}
```


W.r.t. the error message, I know we use `conflicting options` in the `docker` CLI for similar things, e.g.

```
conflicting options: cannot specify both --link and per-network links
```